### PR TITLE
Setup CI

### LINKED ISSUE #98: CI for silkworm
Disable the test tracked by https://github.com/eosnetworkfoundation/silkworm/issues/40 for now to get the tests to pass so that we can enable the CI.

### PR DESCRIPTION
**Pull request by yarkinwho**
And fix some tests.

Resolve #98

### CODE REVIEW

File: .github/workflows/build-node.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```
Comment (arhag) [inline .github/workflows/build-node.sh:1]: Rename file to `build-tests.sh`.

File: .github/workflows/node.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# EOS EVM Silkworm CI
```
Comment (arhag) [inline .github/workflows/node.md:1]: Rename the file to `silkworm.md`.

File: .github/workflows/node.yml [hunk 3]
```diff
@@ -0,0 +1,178 @@
+name: EOS EVM Silkworm CI
```
Comment (arhag) [inline .github/workflows/node.yml:1]: Rename the file to `silkworm.yml`.

**Review by arhag — CHANGES_REQUESTED** [review]

**Review by elmato — APPROVED** [review]

File: .github/workflows/node.md (hunk 2 above)
Comment (yarkinwho) [reply .github/workflows/node.md]: done

File: .github/workflows/node.yml (hunk 3 above)
Comment (yarkinwho) [reply .github/workflows/node.yml]: done

**Review by arhag — APPROVED** [review]

**Pull request merged**